### ISSUE TITLE
Absolute url in schemaLocation attribute gets normalized to base url

### ISSUE THREAD

**Issue by ArthurArslanaliev** [brunato/xmlschema#14]
Lets say I have a schema like this:

```xsd
<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.w3.org/2001/XMLSchema"
            xmlns:platformCore="urn:core_2017_1.platform.webservices.netsuite.com"
            targetNamespace="urn:sales_2017_1.transactions.webservices.netsuite.com" elementFormDefault="qualified">

    <xsd:import namespace="urn:core_2017_1.platform.webservices.netsuite.com"
                schemaLocation="https://webservices.netsuite.com/xsd/platform/v2017_1_0/core.xsd"/>

    <element name="Opportunity" type="platformCore:RecordRef"/>

</xsd:schema>
```
and I'm trying to load it using the following code:

```python
with open('/path_my_schema/schema.xsd', 'r') as f:
    s = XMLSchema(f)
```

the URL value of the `schemaLocation` attribute (https://webservices.netsuite.com/xsd/platform/v2017_1_0/core.xsd) gets normalized to `/path_my_schema/core.xsd` and that raises an error when it tries to fetch schema from remote location.

```
ValueError: unknown url type: '/path_my_schema/core.xsd'
```

I suspect that problem is somewhere in these lines 

https://github.com/brunato/xmlschema/blob/1c660b4dbe6715f01e8c45f39b24646d2f03cc8b/xmlschema/resources.py#L132-L146

My expectation is that if `url` is an absolute url it shouldn't be normalized and needs to be used as is.

**Comment by ArthurArslanaliev** [comment]
@brunato I can make a pr if you confirm that the issue exists.

**Comment by brunato** [comment]
Yes, the problem exists, but I've also changed this code of `normalize_url` in my develop because there was also an issue with Windows paths (C:/...). If you want to make a PR at the bottom there is the current version of `normalize_url` of my "develop".

Thank You

```python
def normalize_url(url, base_url=None):
    """
    Returns a normalized url. If URL scheme is missing the 'file' scheme is set.

    :param url: An relative or absolute URL.
    :param base_url: A reference base URL to join.
    :return: A normalized URL.
    """
    url_parts = urlsplit(url)
    if url_parts.scheme and url_parts.scheme in uses_relative:
        if base_url is None:
            return url_parts.geturl()
        else:
            url_parts = urlsplit(urljoin(base_url, os.path.basename(url_parts.path)))
            return url_parts.geturl()
    elif base_url is None:
        pathname = os.path.abspath(url_parts.geturl())
        return urljoin(u'file:', pathname2url(pathname))
    else:
        base_url_parts = urlsplit(base_url)
        if base_url_parts.scheme and base_url_parts.scheme in uses_relative:
            return urlunsplit((
                base_url_parts.scheme,
                base_url_parts.netloc,
                os.path.join(base_url_parts.path, pathname2url(url)),
                base_url_parts.query,
                base_url_parts.fragment
            ))
        else:
            pathname = os.path.abspath(os.path.join(base_url, url))
            url_parts = urlsplit(pathname2url(pathname))
            if url_parts.scheme and url_parts.scheme in uses_relative:
                return url_parts.geturl()
            else:
                return urljoin(u'file:', url_parts.geturl())
```

**Comment by brunato** [comment]
This bug ought to be fixed in the new version (0.9.12).
